software/messaging/src/main/java/org/apache/brooklyn/entity/zookeeper/ZooKeeperNode.java

```diff
@@ -46,7 +46,7 @@
 
     @SetFromFlag("downloadUrl")
     AttributeSensorAndConfigKey<String, String> DOWNLOAD_URL = ConfigKeys.newSensorAndConfigKeyWithDefault(SoftwareProcess.DOWNLOAD_URL,
-            "http://apache.fastbull.org/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
+            "http://apache.org/dist/zookeeper/zookeeper-${version}/zookeeper-${version}.tar.gz");
```

Comment:
Agree, but we should avoid putting load on apache.org if possible, by mirroring the binary in cloudsoft.com.